wagtail_wordpress_import/functions.py

```diff
@@ -48,3 +48,25 @@ def node_to_dict(node):
     if obj == {"nil": True}:
         return None
     return obj
+
+
+def snakecase_key(key):
+    """
+    Convert the key to snake_case by replacing ':' with '_'
+    and remove any leading '_'
+    This is so the keys are valid as Python kwargs and can be used to filter
+    the page models on on keys in the wp_post_meta which is a JSONField().
+    """
+    return (
+        str(key)  # some keys look like boolean types
+        .replace(":", "_")  # some keys contain ':'
+        .lstrip("_")  # some keys start with '_'
+    )
+
+
+def dict_to_list(node, key):
```

Comment:
**Suggestion** (nitpick): I'd imagine a function called x_to_y to return an object of type y representing the entire input of type x. Here we return an attribute of the input, cast/coerced to type y. A function called get_something sounds closer to what it does.

```suggestion
def get_attr_as_list(node, key):
```

Reply:
Thats a good suggestion